flightdata-kafka-producer/src/main/java/com/jackdaw/producer/FlightCreator.java

```diff
@@ -36,25 +36,25 @@
         if (splitMessage.length != expectedArraySize) {
             return Optional.empty();
         } else {
-            builder.setFlightSymbol(splitMessage[flightSymbolIndex]);
-            builder.setAirline(splitMessage[airlineIndex]);
-            builder.setFlightType(FlightType.valueOf(splitMessage[flightTypeIndex]));
-            builder.setTimeType(TimeType.valueOf(splitMessage[timeTypeIndex]));
-            builder.setTime(splitMessage[timeIndex]);
-            builder.setFlightSituation(FlightSituation.valueOf(splitMessage[flightSituationIndex]));
-            builder.setCodeJustification(splitMessage[codeJustificationIndex]);
-            builder.setOriginAirport(splitMessage[originAirportIndex]);
-            builder.setOriginCity(splitMessage[originCityIndex]);
-            builder.setOriginState(splitMessage[originStateIndex]);
-            builder.setOriginCountry(splitMessage[originCountryIndex]);
-            builder.setDestinationAirport(splitMessage[destinationAirportIndex]);
-            builder.setDestinationCity(splitMessage[destinationCityIndex]);
-            builder.setDestinationState(splitMessage[destinationStateIndex]);
-            builder.setDestinationCountry(splitMessage[destinationCountryIndex]);
-            builder.setDestinationLongitude(NumberUtils.createDouble(splitMessage[destinationLongitudeIndex]));
-            builder.setDestinationLatitude(NumberUtils.createDouble(splitMessage[destinationLatitudeIndex]));
-            builder.setOriginLongitude(NumberUtils.createDouble(splitMessage[originLongitudeIndex]));
-            builder.setOriginAltitude(NumberUtils.createDouble(splitMessage[originAltitudeIndex]));
+            builder.setFlightSymbol(splitMessage[FLIGHT_SYMBOL_INDEX]);
```

Comment:
Nice! Please note, a builder pattern allows to do a fluent style:

```java
builder
  .setFlightSymbol(splitMessage[FLIGHT_SYMBOL_INDEX])
  .setAirline(splitMessage[AIRLINE_INDEX])
  .setFlightType(FlightType.valueOf(splitMessage[FLIGHT_TYPE_INDEX]))
  .setTimeType(TimeType.valueOf(splitMessage[TIME_TYPE_INDEX]))
  .build();
```

It's suggestion, not a blocker 😎